src/python/datastore/data_types.py

```diff
@@ -1131,6 +1131,7 @@ class FuzzStrategyProbability(Model):
   probability_medium_temperature = ndb.FloatProperty()
   probability_high_temperature = ndb.FloatProperty()
   probability_low_temperature = ndb.FloatProperty()
+  engine = ndb.StringProperty()
```

Comment:
I think you need to run `python butler.py generate_datastore_models` to re-generate `src/go/cloud/db/types/types.go` file.